README.md

```diff
@@ -42,14 +42,15 @@ npm i ngx-sub-form
 **Note about the versions:**
 
 | `@angular` version   | `ngx-sub-form` version                               |
-| -------------------- | ---------------------------------------------------- |
+|----------------------|------------------------------------------------------|
 | v <= `7`             | v <= `2.7.1`                                         |
 | `8.x`                | `4.x`                                                |
 | `9.x` <= v <= `12.x` | `5.1.2`                                              |
 | `13.x`               | `5.2.0` (non breaking but new API available as well) |
 | `14.x`               | `6.0.0` (Angular 14 upgrade only)                    |
 | `14.x`               | `7.0.0` (deprecated API is now removed)              |
 | `15.x`               | `8.0.0`                                              |
+| `16.x`               | `9.0.0`                                              |
```

Comment:
I don't understand what's happening on this PR :thinking: 

This change is already in master, how can it show that diff?!

![image](https://github.com/cloudnc/ngx-sub-form/assets/4950209/442a5a98-6a52-4ec0-9f37-fca7e81326eb)

Reply:
Can you please try to rebase from master because I really don't understand what's going on with that MR and the changes that it shows :o How can I get a diff if it's already there?!

Reply:
I wasn't sure if you were going to revert my initial PR. I've now rebased my pull request on top of master. The above change is no longer visible.

Reply:
Ah I see, thanks :+1: